### PR TITLE
Fix truncated artifacts cache

### PR DESCRIPTION
**Pull request by BrunoBonacci**
it fixes https://github.com/clojure-emacs/refactor-nrepl/issues/284

- [?] The commits are consistent with our [contribution guidelines](CONTRIBUTING.md)
- [Y] You've added tests (if possible) to cover your change(s)
- [Y] All tests are passing (run `lein do clean, test`)
- [Y] Code inlining with mranderson works and tests pass with inlined code (run `./build.sh install` -- takes a long time)
- [N] You've updated the changelog (if adding/changing user-visible functionality)
- [N] You've updated the readme (if adding/changing user-visible functionality)

### CODE REVIEW

**Comment by BrunoBonacci** [comment]
Test failures were present prior to my change!

File: src/refactor_nrepl/artifacts.clj [hunk 1]
```diff
@@ -115,12 +115,18 @@
...
+            *print-level*  nil]
+    (prn-str v)))
+
 (defn- update-artifact-cache!
   []
   (let [clojars-artifacts (future (get-artifacts-from-clojars!))
         maven-artifacts (future (get-artifacts-from-mvn-central!))]
     (reset! artifacts (into @clojars-artifacts @maven-artifacts))
-    (spit artifacts-file @artifacts)
+    (spit artifacts-file (prn-str-full @artifacts))
```
Comment (bbatsov) [inline src/refactor_nrepl/artifacts.clj]: Why don't you just add the binding here? I think it's going to be much easier to figure out what's going on this way.

**Comment by BrunoBonacci** [comment]
@bbatsov is there any other change you want me to push before you merge this PR??

**Comment by BrunoBonacci** [comment]
@bbatsov  I was hoping you could include this fix in the `2.5.0`  :-(

I've rebased the changes on your latest changes now. Please do let me know if there is any other change I should apply before you merge it.

**Comment by bbatsov** [comment]
I'm really sorry about this! Somehow I missed this PR when I was preparing for the release yesterday. :-(

**Pull request merged**